checkov/common/util/ext_argument_parser.py

```diff
@@ -468,6 +468,13 @@ def add_parser_args(self) -> None:
                  "information on allowed filters. Format: policy.label=test,cloud.type=aws ",
             default=None,
         )
+        self.add(
+            "--policy-metadata-filter-exception",
+            help="comma separated key:value string to exclude filtered policies based on Prisma Cloud policy metadata. "
```

Comment:
Can you add a description of what happens when you use both options (the subtraction logic), to this as well as to the other one?

Reply:
Done ✅